src/components/app/timeIntervalDropdown/timeIntervalDropdown.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React, { Component } from 'react';
+
```

Comment:
are we leaving extra lines between imports to signify different types of import? May be we should get into habit of adding //in the beginning of section of imports?

Reply:
Yes, at least if there are several of them. See the [guidelines for imports](https://github.com/Azure/pcs-remote-monitoring-webui/wiki/Code-Guide#imports).